### ISSUE TITLE
External Power Source Cannot be used in the Energy Panel

### ISSUE THREAD

**Issue by brianvg** [MrIceman11/e3dc-homeassistant#9]
The value given by the sensor.e3dc_external_power is a negative number. To be able to use it in the energy panel, the sensor should be turned into an absolute value. Suggest adding this to sensors.yaml.

**Comment by MrIceman11** [comment]
Hi, here is the code to convert. Could you test it if it works? Just paste the code into the sensors.yaml file.

    - platform: template
      sensors:
        e3dc_external_power:
          friendly_name: E3DC External Power
          unit_of_measurement: W
          value_template: >
            {{ states('sensor.e3dc_external_power') | float | abs }}

**Comment by brianvg** [comment]
Hello!
Yes, I already implemented something like that. I did not realize the same sensor name "e3dc_external_power" could be used for both sensors, so I created a new one called e3dc_external_solar_power, since that corresponds to my use case. Otherwise it's identical, and I can confirm it works. I was able to add the integrator to the energy panel after that (in my case I had to change the integrator as well).
